home-data-migration/src/main/resources/db/changelog/0.0.1/Issue156.xml

```diff
@@ -0,0 +1,100 @@
+            <column name="id" value="21"/>
+            <column name="name" value="CREATE_COOP_CONTACT_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permissions">
+            <column name="id" value="22"/>
+            <column name="name" value="GET_COOP_CONTACT_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permissions">
+            <column name="id" value="23"/>
+            <column name="name" value="GET_ALL_COOP_CONTACT_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permissions">
+            <column name="id" value="24"/>
+            <column name="name" value="DELETE_COOP_CONTACT_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permissions">
+            <column name="id" value="25"/>
+            <column name="name" value="UPDATE_COOP_CONTACT_PERMISSION"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue156.003" author="likeRewca-Yura">
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="20"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="21"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="22"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="23"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="24"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role_permissions">
+            <column name="role_id" value="1"/>
+            <column name="permission_id" value="25"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue156.004" author="likeRewca-Yura">
+        <addUniqueConstraint schemaName="public"
+                             tableName="users"
+                             constraintName="user_email_key"
+                             columnNames="email"/>
+    </changeSet>
+
+</databaseChangeLog>
```

Comment:
Please do not forget to add newline at the end of the file.

Reply:
yeap, I forgot about this